ui/src/authentication/services/SessionManager.js

```diff
@@ -8,6 +8,10 @@ class SessionManager {
     return this.user === null;
   }
 
+  isAuthenticated() {
```

Comment:
fixes some leftovers after SPA login refactoring

Reply:
This was already done in @topless PR, right?